src/pages/Lobby.tsx

```diff
@@ -311,7 +311,7 @@ const LobbyPage = ({ ws, currentUser }: LobbyProps): JSX.Element => {
         >
           <h2 className="">Games</h2>
           {games.map((game) => (
-            <GameRow gameId={game.gameId} onClickHandler={requestJoinGame} roomName={game.gameName} />
+            <GameRow gameId={game.gameId} onClickHandler={requestJoinGame} roomName={game.gameName} key={game.gameId} />
```

Comment:
Good catch. Saw this error on console but didn't really think too much of it

Reply:
and this error actually introduced a small bug that u might see before:
user leaves a game by clicking on the Leave Game button, all the games in the game list are gone.